aQute.libg/src/aQute/lib/home/Home.java

```diff
@@ -0,0 +1,21 @@
+package aQute.lib.home;
+
+public class Home {
+	public static final String	USER_HOME_BND_DEFAULT	= "~/.bnd";
+	public static final String	USER_HOME_BND_ENV		= "bnd.home.dir";
+
+	public static final String BND_USER_HOME() {
```

Comment:
The all caps method name is not typical. All caps is generally only used to constants.